dbt/include/presto/macros/materializations/table.sql

```diff
@@ -0,0 +1,61 @@
+{% materialization table, adapter='presto' %}
+  {%- set identifier = model['alias'] -%}
+  {%- set tmp_identifier = model['name'] + '__dbt_tmp' -%}
+  {%- set backup_identifier = model['name'] + '__dbt_backup' -%}
+
+  {%- set old_relation = adapter.get_relation(database=database, schema=schema, identifier=identifier) -%}
+  {%- set target_relation = api.Relation.create(identifier=identifier,
+                                                schema=schema,
+                                                database=database,
+                                                type='table') -%}
+  {%- set intermediate_relation = api.Relation.create(identifier=tmp_identifier,
+                                                      schema=schema,
+                                                      database=database,
+                                                      type='table') -%}
+
+  /*
+      See ../view/view.sql for more information about this relation.
```

Comment:
```suggestion
      See ../view.sql for more information about this relation.
```

Reply:
Well spotted. I'm just going to remove this comment entirely